src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -20,7 +20,7 @@ import { getReportUrl } from "../../../utilities/report-utils";
 
 const kDefaultHeight = 300;
 
-const kInteractiveStateRequestTimeout = 2000; // ms
+const kInteractiveStateRequestTimeout = 20000; // ms
```

Comment:
This seems too long to me. If a student is waiting 20s without any UI feedback after hitting the next button they are going to think something is broken. 2s seems much better. 

I know why this is bumped up, but I think we have to add some kind of feedback. I added a new story for this: https://www.pivotaltracker.com/story/show/181511767 so it doesn't need to hold up this PR.

Reply:
Just for documentation - this was changed to align it with the other two 20 second timeouts:

- the firebase request tracker: https://github.com/concord-consortium/activity-player/blob/master/src%2Ffirebase-db.ts#L71
- the ajax request tracker: https://github.com/concord-consortium/activity-player/blob/master/src%2Futilities%2Frequest-tracker.ts#L1 (this is the default, the callers use the Firebase 20 second timeout)

If these are not aligned the user can switch pages and unload the interactive before the request finishes which then causes an unrelated error message to show on the new page when the 20 second timeout fails.

Reply:
> Just for documentation - this was changed to align it with the other two 20 second timeouts:
> 
> * the firebase request tracker: https://github.com/concord-consortium/activity-player/blob/master/src%2Ffirebase-db.ts#L71
> * the ajax request tracker: https://github.com/concord-consortium/activity-player/blob/master/src%2Futilities%2Frequest-tracker.ts#L1 (this is the default, the callers use the Firebase 20 second timeout)
> 
> If these are not aligned the user can switch pages and unload the interactive before the request finishes which then causes an unrelated error message to show on the new page when the 20 second timeout fails.

Seems like that comment should be in the code.